.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
This template is very similar to the previous one (`bug-report.md`), but has less information. Can it be removed?